libs/application/templates/children-residence-change/src/fields/Overview/index.tsx

```diff
@@ -159,9 +158,7 @@ const Overview = ({
           text={m.contract.general.description}
           format={{
             otherParent:
-              parentKey === Roles.ParentA
-                ? parentB.fullName
-                : applicant.fullName,
+              role === Roles.ParentA ? parentB.fullName : applicant.fullName,
```

Comment:
Since we are only using this in one place should we join the logic here and on line 87 and change line 87 to:
`const otherParentName = application.state === 'draft' ? parentB.fullName : application.fullName` ?